### ISSUE TITLE
Expand: moeten alle geledingen van de expandparameter worden geexpandeerd of alleen de laatste?

### ISSUE THREAD

**Issue by johannesbattjes** [VNG-Realisatie/gemma-zaken#2398]
Stel: expand=relevanteAndereZaken.status
Wordt dan alleen status geexpandeerd of ook relevanteAndereZaken?
Waar staat dit beschreven?

@HenriKorver

**Comment by heuvea** [comment]
Lijkt erop dat met één pad aanduiding (relevanteAndereZaken.status) zowel de zaak als status wordt expanded
![relevanteanderezaak_status_expand](https://github.com/VNG-Realisatie/gemma-zaken/assets/66989902/c8711381-b766-41e5-a521-31cb27871a89)

**Comment by joeribekker** [comment]
Dit lijkt mij ook juist maar moet inderdaad omschreven staan ergens.

**Comment by HenriKorver** [comment]
Klopt, bij geneste expand worden alle velden die in het pad (met punt-notatie) voorkomen geëxpandeerd. Ik zal ervoor zorgen dat dit duidelijk wordt beschreven in de specificatie.
